cookie-manager.js

```diff
@@ -357,9 +375,21 @@
         }
 
         const theBanner = document.getElementById(options['cookie-banner-id']);
-        const acceptAllButton = document.querySelector('button[type="submit"]');
         if (theBanner !== null) {
-            addAcceptAllListener(acceptAllButton);
+            const cookieButtons = theBanner.querySelectorAll('button');
+            let acceptAllButton, rejectAllButton;
+            for (let button of cookieButtons) {
```

Comment:
We may need to refactor `for... of`. It is not compatible with IE11 ([GDS Browser support list](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices)).

Just a `for` loop may be more compatible.